lib/cli/interactive-setup/console-dev-mode.js

```diff
@@ -0,0 +1,337 @@
+'use strict';
+
+const { writeText, style } = require('@serverless/utils/log');
+const WebSocket = require('ws');
+const chalk = require('chalk');
+const resolveAuthMode = require('@serverless/utils/auth/resolve-mode');
+const streamBuffers = require('stream-buffers');
+const apiRequest = require('@serverless/utils/api-request');
+
+const stage = process.env.SERVERLESS_PLATFORM_STAGE || 'prod';
+const socketUrl =
+  stage === 'prod'
+    ? 'wss://ehst4ktjyi.execute-api.us-east-1.amazonaws.com/prod'
+    : 'wss://4c684ym73e.execute-api.us-east-1.amazonaws.com/dev';
+
+const streamBuff = new streamBuffers.ReadableStreamBuffer({
+  frequency: 500,
+  chunkSize: 2048 * 1000000,
+});
+
+const getDuration = (startTime, endTime) => {
+  return new Date(endTime).getTime() - new Date(startTime).getTime();
+};
+
+const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+
+const formatDuration = (milliseconds) => {
+  const seconds = milliseconds / 1000;
```

Comment:
For all of this "UI" related stuff I am thinking I should move this out into a package that we can share between console and the CLI, possibly even set up so we can edit these configs remotely but I need to do a bit more research before I can move on this idea 👍 

So for the most part, just ignore these "UI" related utilities for now. I just didn't want this small detail to block reviews and stuff.

Reply:
I am going to move this to `@serverless/utils` 👍